saos-batch-jobs/src/test/java/pl/edu/icm/saos/batch/jobs/importer/CcJudgmentRemoveObsoleteJobTest.java

```diff
@@ -0,0 +1,150 @@
+package pl.edu.icm.saos.batch.jobs.importer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobExecution;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.google.common.collect.Lists;
+
+import pl.edu.icm.saos.batch.core.JobForcingExecutor;
+import pl.edu.icm.saos.batch.jobs.BatchJobsTestSupport;
+import pl.edu.icm.saos.batch.jobs.JobExecutionAssertUtils;
+import pl.edu.icm.saos.common.testcommon.category.SlowTest;
+import pl.edu.icm.saos.importer.commoncourt.judgment.download.SourceCcjExternalRepository;
+import pl.edu.icm.saos.importer.commoncourt.judgment.remove.SourceCcRemovedJudgmentsFinder;
+import pl.edu.icm.saos.persistence.common.TestPersistenceObjectFactory;
+import pl.edu.icm.saos.persistence.enrichment.EnrichmentTagRepository;
+import pl.edu.icm.saos.persistence.model.CommonCourtJudgment;
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.model.RemovedJudgment;
+import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+import pl.edu.icm.saos.persistence.repository.RemovedJudgmentRepository;
+
+/**
+ * @author madryk
+ */
+@Category(SlowTest.class)
+public class CcJudgmentRemoveObsoleteJobTest extends BatchJobsTestSupport {
+
+    @Autowired
+    private Job ccJudgmentRemoveObsoleteJob;
+    
+    @Autowired
+    private JobForcingExecutor jobExecutor;
+    
+    
+    @Autowired
+    private JudgmentRepository judgmentRepository;
+    
+    @Autowired
+    private RemovedJudgmentRepository removedJudgmentRepository;
+    
+    @Autowired
+    private EnrichmentTagRepository enrichmentTagRepository;
+    
+    @Autowired
+    private SourceCcRemovedJudgmentsFinder sourceCcRemovedJudgmentsFinder;
+    
+    
+    @Autowired
+    private TestPersistenceObjectFactory testPersistenceObjectFactory;
+    
+    
+    private final static int JUDGMENTS_COUNT = 15;
+    private final static int REMOVED_COUNT = 3;
+    
+    private Judgment judgmentToRemove1;
+    private Judgment judgmentToRemove2;
+    private Judgment judgmentToRemove3;
+    
+    @Before
+    public void setup() {
```

Comment:
You use @before to create data used by one specific test. @Before should be used to prepare test env not specific for one test.
That's why you have to link the data with the external consts.